src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -10543,17 +10543,6 @@ def rolling_count(
     ) -> None:
         ErrorMessage.method_not_implemented_error(name="count", class_="Rolling")
 
-    def rolling_sem(
```

Comment:
Same, not clear why this was moved down.

Reply:
Ditto above